lib/routes/routeBackbeat.js

```diff
@@ -105,6 +110,7 @@ function putData(request, response, bucketInfo, objMd, log, callback) {
                 dataStoreName,
                 size: payloadLen,
                 start: 0,
+                dataStoreETag: `${partNumber}:${md5}`,
```

Comment:
I don't see the value of sending partnumber and md5 from backbeat only to be sent back to backbeat. We can just construct this in BB instead.